src/pc/1.14/ChunkColumn.js

```diff
@@ -197,6 +199,14 @@ module.exports = (Block, mcData) => {
       return this.sectionMask
     }
 
+    getMaskArray () {
```

Comment:
is this new method useful ?

Reply:
No, it's a hold over from when I was rebasing from Archengius. Will remove

Reply:
Not new, present in 1.16

Reply:
I don't think it's useful either, but again I just Ctrl+C, Ctrl-V

Reply:
https://github.com/PrismarineJS/prismarine-chunk/pull/133/files pretty sure it was added in this pr in all versions